src/FirstSteps/RPG/Heroes/Hero.cs

```diff
@@ -100,11 +104,29 @@ public HeroModel ToHeroModel()
                     {
                         Name = item.Name,
                         Price = item.Price,
-                        Weight = item.Weight
+                        Weight = item.Weight,
                     }).ToList()
-                }
+                },
             };
             return heroModel;
+
+        }
+        public bool Resurrect()
```

Comment:
Your version is okay, but look at this alternative where the code has fewer nestings:
 ```csharp
 public bool Resurrect()
{
    if(_coinsBag < 5)
    {
        Display.ErrorText("You dont have enought coins.");
        return false;
    }

    int healthReturned = _maxHealth / 2;
    _health = healthReturned;
    _coinsBag -= 5;
    
    Display.DefaultText($"You returned {healthReturned} health.");
    return true;
}
```